CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+##2018-01-23 - Supported Release 1.3.0
+###Summary
+
+Summary.
+
+###Features
+* Add a [Puppet Task](https://puppet.com/docs/bolt/0.x/running_tasks_and_plans_with_bolt.html#concept-1376) for performing on-demand reboots ([MODULES5804](https://tickets.puppetlabs.com/browse/MODULES-5804))
+* Add capability to reboot a Windows machine if [specific conditions](https://github.com/puppetlabs/puppetlabs-reboot#onlyif) are met ([MODULES-4328](https://tickets.puppetlabs.com/browse/MODULES-4328))
+
```

Comment:
Adding puppet 5 is a feature.

Reply:
Adding `unless` parameter is also a new feature interface-wise and isn't mentioned.